src/consensus/aft/raft.h

```diff
@@ -1167,6 +1174,24 @@ namespace aft
         r.idx,
         r.prev_idx);
 
+      if (is_new_follower)
+      {
+        if (state->last_idx != r.prev_idx)
```

Comment:
If this node was a lagging follower in the previous term and is told by the new primary about `r.prev_idx`, which is in the future (`r.prev_idx > state->last_idx`), the we would rollback in the future? I'm assuming this is fine as all actions performed by `rollback()` have no effect in this case, but I wonder if the check here should be `state->last_idx > r.prev_idx` instead?

Reply:
Changed this to `state->last_idx > r.prev_idx`.